src/main/java/nl/fuchsia/controller/FeitContoller.java

```diff
@@ -0,0 +1,33 @@
+package nl.fuchsia.controller;
+
+import nl.fuchsia.model.Feit;
+import nl.fuchsia.services.FeitService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/feiten")
+public class FeitContoller {
+    private final FeitService feitService;
+
+    public FeitContoller(FeitService feitService) {
+        this.feitService = feitService;
+    }
+
+    /**
+     * Deze methode voegt een feit aan de database toe
+     * @param feit wordt uit een Json post opgehaald
```

Comment:
@Param. hier ontbreekt dat er een validatie plaats vindt @Validate en dat het request een JSON object is. @RequestBody.